packages/@sanity/studio-hints/package.json

```diff
@@ -25,10 +25,7 @@
   ],
   "dependencies": {
     "@sanity/block-content-to-react": "^2.0.7",
-    "lodash": "^4.17.15",
-    "next": "^9.1.3",
     "react-copy-to-clipboard": "^5.0.2",
-    "react-icons": "^2.2.7",
```

Comment:
Two files in this package is using react-icons:
```js
// src/components/ToggleSidecarButton.js
import HelpOutline from 'react-icons/lib/io/ios-help-outline'
import Exit from 'react-icons/lib/io/ios-close-outline'

// src/components/serializers/internalLinkSerializer.js
import DocsIcon from 'react-icons/lib/fa/book'
```

We should either keep `react-icons` as a dependency, or move these icons to be parts and import them from there. Moving this to a peer dependency means each studio using the package should declare it as a dependency, which we don't want.